mscore/preferences.cpp

```diff
@@ -432,6 +437,10 @@ void Preferences::read()
       alsaFragments      = s.value("alsaFragments", alsaFragments).toInt();
       portaudioDevice    = s.value("portaudioDevice", portaudioDevice).toInt();
       portMidiInput      = s.value("portMidiInput", portMidiInput).toString();
+      portMidiOutput     = s.value("portMidiOutput", portMidiOutput).toString();
+      //portMidiInputBufferCount = s.value("portMidiInputBufferCount", portMidiInputBufferCount).toString();
+      //portMidiOutputBufferCount = s.value("portMidiOutputBufferCount", portMidiOutputBufferCount).toString();
```

Comment:
I couldn't decide whether to expose the buffer size to the prefs GUI or not.  I actually made the output buffer count default extra big, so can handle large scores and large latencies, but it might be unnecessarily large, so I might need to experiment with finding the smallest size that works for the most dense score that is practical.